easybuild/easyblocks/c/clang_aomp.py

```diff
@@ -169,41 +185,48 @@ def sanity_check_step(self):
             arch = 'aarch64'
         else:
             print_warning("Unknown CPU architecture (%s) for OpenMP offloading!" % arch)
-        custom_paths['files'].append('lib/libomptarget.rtl.%s.%s' % (arch, shlib_ext))
+
+        custom_paths['files'].append(os.path.join('lib', 'libomptarget.rtl.%s.%s' % (arch, shlib_ext)))
+
         # If CUDA offloading support was requested, check that correct omptarget was built
         if get_software_root('CUDA'):
-            custom_paths['files'].append('lib/libomptarget.rtl.cuda.%s' % shlib_ext)
+            custom_paths['files'].append(os.path.join('lib', 'libomptarget.rtl.cuda.%s' % shlib_ext))
+
             for arch in self.cuda_archs:
                 sm_arch = 'sm_%s' % arch
-                custom_paths['files'].append('lib/libdevice/libomptarget-nvptx-%s' % sm_arch)
-                custom_paths['files'].append('lib/libdevice/libaompextras-nvptx-%s' % sm_arch)
-                custom_paths['files'].append('lib/libdevice/libm-nvptx-%s' % sm_arch)
```

Comment:
@nordmoen It seems like this wasn't correct already: there's no `lib/libdevice/libomptarget-nvptx-sm_70` installed at all when `CUDA` is added as a build dep.

Should this just check for `lib/libomptarget-nvptx-sm_70.bc`?

`lib/libdevice/libm-nvptx-sm_70.bc` is there too, but is always there it seems?

Reply:
Need to test a bit more on my end